src/app/shared/feature-resolver/feature-resolver.service.ts

```diff
@@ -1,15 +1,17 @@
 import { Injectable } from '@angular/core';
-import { ApplicationFeature, FeatureState, FeatureStateResolver } from '@hypertrace/common';
+import { ApplicationFeature, FeatureStateResolver, FeatureFlagValue } from '@hypertrace/common';
 import { Observable, of } from 'rxjs';
 
 @Injectable()
 export class FeatureResolverService extends FeatureStateResolver {
-  public getFeatureState(flag: string): Observable<FeatureState> {
-    switch (flag) {
+  public getFeatureFlagValue<T extends FeatureFlagValue = FeatureFlagValue>(feature: string): Observable<T> {
+    switch (feature) {
       case ApplicationFeature.PageTimeRange:
-        return of(FeatureState.Disabled);
+        return of(false as T);
+      case ApplicationFeature.FeatureDefaultTimeRange:
+        return of(({ 'path-1': '1h', 'path-2': '1d' } as unknown) as T); //dummy data
```

Comment:
Should default to empty. These are really just overrides to allow config to override what's in code.